ConferenceService/Models/Conference.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace ConferenceService.Models
+{
+    using MongoDB.Bson.Serialization.Attributes;
+    using MongoDB.Bson;
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Data model representing a conference.
+    /// </summary>
+    public class Conference
+    {
+        /// <summary>
+        /// A unique ID for each conference.
+        /// </summary>
+        [BsonId]
+        [BsonRepresentation(BsonType.ObjectId)]
+        public string Id { get; set; }
+
+        /// <summary>
+        /// The name of the conference.
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Conference start date.
+        /// </summary>
+        public DateTime StartDate { get; set; }
```

Comment:
Add the [BsonDateTimeOptions] attribute to gain control on datetime serialization :)

Reply:
Will fix!